src/content-service/models/venue-address.ts

```diff
@@ -10,26 +11,34 @@ export class VenueAddress {
   private readonly postCode: string;
   private readonly region: Region;
   private readonly country: Country;
+  private readonly latitude: string;
+  private readonly longitude: string;
 
-  constructor (addressData: ApiAddressData) {
+  constructor (addressData: (ApiAddressData | VenueApiAddressData)) {
```

Comment:
What is the difference between those two types and why can't we just use VenueApiAddressData?

Reply:
Ok, but this model is located in the content-service folder. I don't think you should use this model in venue service.

Reply:
done